build/pmm-client-docker/Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi7/ubi
 RUN yum -y install iproute nc \
     && rm -rf /var/cache/yum
 
 COPY pmm2-client.tar.gz /tmp/pmm2-client.tar.gz
+RUN groupadd -g 1002 pmm-agent
+RUN useradd -u 1002 -r -g pmm-agent -s /sbin/nologin \
+            -d /usr/local/percona/pmm2 \
+            -c "Default Application User" pmm-agent
+
+RUN install -d -o pmm-agent -g pmm-agent -m 0775 /usr/local/percona/pmm2 \
+    && install -d -o pmm-agent -g pmm-agent -m 0775 /run /var/log /etc/rc.d/init.d
```

Comment:
I don't think we will use any of those directories: we don't put anything into `/run`, logging will use stderr, and there is no init.d script being used.